### ISSUE TITLE
This theme no longer working with latest bean

### ISSUE THREAD

**Issue by tonamphong** [ThemeButler/tm-site#1]
Hi there,

I'm testing Bean framework and faced the problem with Bean latest version and tm-site theme. Could you please check this case? Is it working fine with the latest Bean?

**Comment by tonamphong** [comment]
okay, just fixed the problem :)

**Comment by smokeyfro** [comment]
Hi ToNamPhong,

Sorry for the delayed response - I have had my hands full lately. The tm-site theme is the live WIP theme used on ThemeButler.com and is intended as a practical example of using Beans on an actual site. While you're welcome to take the site and adapt it to your needs, I can't guarantee that it will always work.

Out of curiosity, what was the issue in the end?

**Comment by tonamphong** [comment]
I got this error message before 
"Parse error: syntax error, unexpected end of file in /var/www/test.com/public_html/ck/wp-content/themes/tm-site/functions.php on line 219"

My test case: Wordpress site with dummy content, beans latest and tm-site child theme.

This caused by your ads footer function line 206, just changed

<? }
to
<?php }
and its okay now.

Your child theme and custom plugin are GREAT.  I have learned a lot from your custom code and snippets.
Keep up the good work devignerforhire :+1:

**Comment by smokeyfro** [comment]
Ah right :) Thanks for the heads up! Glad to hear you're finding it useful! Hearing comments like that really motivates me to do more, so thanks for that! I have lots more coming ;-) Did you see the new theme I am working on? http://www.themebutler.com/2015/10/voyager-sneak-peek/

**Comment by tonamphong** [comment]
Yes, I see. Really nice. We (me and Beans fan) cant wait to see the child-themes market for Beans ;)

**Comment by paaljoachim** [comment]
Hey ToNamPhong
@ToNamPhong 

Can you share the fixed theme? As I am getting a page full of errors. Thanks.
